src/languages/es-ES.ts

```diff
@@ -303,6 +325,30 @@ export default class extends Language {
 		COMMAND_PAUSE_DESCRIPTION: `Pausa la canción actual.`,
 		COMMAND_PAUSE_SUCCESS: '⏸ Pausado.',
 		COMMAND_PLAY_DESCRIPTION: `¡Empecemos la cola!`,
+		COMMAND_PLAY_EXTENDED: builder.display('play', {
+			extendedHelp: [
+				`Ponga música en cola y permítame comenzar a improvisar para su disfrute.
+				Cuando use este comando, me uniré automáticamente a su canal de voz y comenzaré a reproducir la primera canción en mi cola.
```

Comment:
"mi cola" sounds really weird in Spanish as it's often referred to "my d\*\*k".
```suggestion
				Cuando use este comando, me uniré automáticamente a su canal de voz y comenzaré a reproducir la primera canción en la cola.
```